programs.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: page
+title: Programs
+permalink: /programs/
+nav_order: 1
+---
+
+Here you can find programs for some popular sequences. Links to the complete program lists can be found at the bottom of this page. There are currently more than 100,000 programs for integer sequences available.
```

Comment:
Replace `Here you can find programs for some popular sequences.` with `Here are programs for some popular integer sequences.`